CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Pending
+
+* Explicitly support Rails 8.0
```

Comment:
I don't think this is worth an actual release but trying to be proactive about updating this